README.md

```diff
@@ -3791,7 +3789,7 @@ type MyType = MyTuple[2]; // boolean
 
 Several built-in utility types can be used to manipulate types, below a list of the most common used:
 
-#### Awaited<T>
+#### Awaited\<T\>
```

Comment:
Have you tried to use pandoc on the Chinese version? I suspect we could have the same issue there.

Reply:
I didn't try that on the Chinese version

Reply:
I confirm is working. Thanks for your support!